src/UI/images/Image.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import styles from '../../styles.scss';
+
+export default function Image({ image }) {
```

Comment:
```suggestion
export default function Image({ image, ...props }) {
```
it's important we pass props like this and then to the jsx in case we want to pass another prop in the future when using the component